apps/AEPSampleApp/extensions/Core.js

```diff
@@ -1,6 +1,7 @@
 import React, {Component} from 'react';
 import {Button, Platform, StyleSheet, Text, View, ScrollView, NativeModules} from 'react-native';
-import {AEPCore, AEPLifecycle, AEPSignal, AEPMobileLogLevel, AEPMobilePrivacyStatus, AEPMobileVisitorAuthenticationState, AEPVisitorID, AEPExtensionEvent} from '@adobe/react-native-aepcore';
+import {AEPIdentity as AEPIdentity, AEPCore, AEPLifecycle, AEPSignal, AEPMobileLogLevel, AEPMobilePrivacyStatus, AEPMobileVisitorAuthenticationState, AEPVisitorID, AEPExtensionEvent } from '@adobe/react-native-aepcore';
+import {AEPIdentity as AEPEdgeIdentity } from '@adobe/react-native-aepedgeidentity';
```

Comment:
Should we add sample for core Identity and Edge Identity here?

Reply:
We can have a separate view for Edge Identity. 
It might be a good idea to make this sample app similar with the Swift Sample app in how the extensions are demoed and split in separate views.

Reply:
I have a separate view for Edge Identity.  But wondering if need to give an example how user can import when have both identity in one view.

Reply:
![Screen Shot 2021-09-01 at 9 26 46 AM](https://user-images.githubusercontent.com/29806547/131713165-59d9cfe6-3c74-4ff6-b603-4c31b25071b7.png)

Reply:
Core API View
![coreAPIView](https://user-images.githubusercontent.com/29806547/131713531-61281637-ad8d-4d91-8723-0ea7ec7fe467.png)

Reply:
Edge Identity View
![edgeIdentityview](https://user-images.githubusercontent.com/29806547/131713575-8d29fce6-6a34-4b32-8d01-a8055aca7724.png)

Reply:
I think we can keep the example with importing both.

Reply:
I removed import Edge Identity from Core view.
Add an EdgeIdentityAndIdentity.js file for both Identities case.